msg/FlapsSetpoint.msg

```diff
@@ -0,0 +1,3 @@
+uint64 timestamp        		# time since system start (microseconds)
```

Comment:
We should merged these common types (the actual .msg) and then have different topics for ORB_ID(flaps_setpoint), ORB_ID(spoilers_setpoint), etc.

Something like ActuatorSetpoint.msg or NormalizedActuatorSetpoint.msg?

Reply:
But it's not directly an actuator setpoint - e.g. spoiler_setpoint.normalized_setpoint can be mapped to ailerons. Calling it "Actuator" something sounds confusion to me.
Agree that we could combine the Flaps and Spoiler "controls" topics, just need to find a good name. AerodynamicDevicesConfiguration.msg, with FlapsConfiguration and SpoilerConfiguration? Or Configuration replaced with Setpoint? Or back to Controls?

Reply:
Or even just NormalizedSetpoint.msg could be fine, the main point is to share these common data types (and get away from .msg === topic).

Reply:
Will make the change and rebase on latest main after the L1 purge is through.

Reply:
We'll test when it's done (flying tomorrow but pretty much every few days so no rush for tomorrow).